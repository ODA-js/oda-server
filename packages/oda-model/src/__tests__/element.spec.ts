import 'jest';
import { Element } from '..';

describe('element', () => {
  it('loads', () => {
    const res = new Element({
      metadata: { some: 'data' },
    });
    expect(res).toMatchSnapshot('raw');
    expect(res.toObject()).toMatchSnapshot('toObject');
  });
});
