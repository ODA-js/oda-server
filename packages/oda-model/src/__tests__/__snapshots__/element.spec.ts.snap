// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`element loads: raw 1`] = `
Element {
  "$obj": Object {},
  "metadata_": Object {
    "some": "data",
  },
}
`;

exports[`element loads: toObject 1`] = `
Object {
  "metadata": Object {
    "some": "data",
  },
}
`;
